code/game/objects/items/weapons/tools/_tools.dm

```diff
@@ -863,10 +863,13 @@
 	return TRUE
 
 //Returns the amount of fuel in tool
-/obj/item/proc/get_fuel()
+/obj/item/tool/proc/get_fuel()
```

Comment:
REALLY dont do this one

Reply:
Neither of those are used on anything that isn't a tool, unless ctrl-shift-f lied to me.

Reply:
Actually no, I'm dumb. I'll fix that, there's a better way.